TAC/Meetings/2022/2022-10-06/CCC-TAC-Minutes-2022-10-06.md

```diff
@@ -0,0 +1,149 @@
+
+### Welcome
+   * Dave Thaler (DT) opened the call at 7:05am PT
+   * DT welcomed the members of the TAC and reviewed the values of the CCC and the antitrust policy of the Linux Foundation
+   * Kurt Taylor (KT) captured the meeting minutes
+   * DT reviewed the agenda
+
+### Roll Call / Attendance
+
+***Voting members of the TAC***
+
+*Per the [charter](https://charter.confidentialcomputing.io), all [CCC Premier members](https://confidentialcomputing.io/members/) receive one vote on the TAC. Quorum for votes is at least 50% of voting members present.*
+
+   * Accenture - Giuseppe Giordano
+   * Ant Group - Hongliang Tian (Tate)
+   * Arm - Thomas Fossati / Michael Lu
+   * Google - Cfir Cohen + / Catalin Sandu
+   * Huawei - Zhipeng (Howard) Huang +
+   * Intel - Dan Middleton + / Simon Johnson
+   * Meta - Eric Northup + / Shankaran Gnanashanmugam
+   * Microsoft - Dave Thaler (TAC chair) +
+   * Red Hat - Lily Sturmann  / Dimitrios Pendarakis
+
+   " + " indicates attending
+
+***Other Attendees***
+
+   * Alec Fernandez (Microsoft)
+   * Eric Voit (Cisco)
+   * Francois-Xavier Marseille (THALES)
+   * Jethro Beekman (Fortanix)
+   * Matthieu Legre (Cysec)
+   * Naveen Cherukuri (NVIDIA)
+   * Nick Vidal (Profian)
+   * Penglin Yang (China Mobile)
+   * Xinxin Fan (IoTex)
+   * Mark Novak (JP Morgan Chase)
+   * Ahmad Atamli (NVIDIA)
+   * Alec Fernandez (Microsoft)
+   * Andy Dellow (Huawei UK)
+   * Francois-Xavier Merseille (Thales)
+   * Henk Birkholz (Fraunhofer SIT)
+   * Ionut Mihalcea (ARM)
+   * Suresh Sugamar (TII)
+   * Ravi Sahita (Rivos)
+   * Drasko Draskovic (Ultraviolet)
+   * Helen Lau (Linux Foundation)
+   * Kurt Taylor (Linux Foundation)
+
+**TAC Tech Talks**
+
+* Andy Dellow - HC / RISC-V Security
+   * Capability Hardware Enhanced RISC Instructions (CHERI) https://www.cl.cam.ac.uk/research/security/ctsrd/cheri/
+   * email: security@lists.riscv.org
+* Ravi Sahita - RVI Sec/TC-SIG/AP-TEE TG (Trusted Computing SIG in Application Process in Trusted Execution Environments) Task Group
+   * AP-TEE specification Github - https://github.com/riscv-non-isa/riscv-ap-tee/tree/main/specification
+   * Linux Plumbers discussion on AP-TEE - https://lpc.events/event/16/contributions/1167/attachments/1069/2071/LPC2022_Sahita_Conf_Comp_on_RISCV.pdf
+   * Salus (TSM) Github - https://github.com/rivosinc/salus
+   * Platform Security Model Specification Github - https://github.com/riscv-non-isa/riscv-platform-security-model
+* ACTION: [Lily] recommendation is to make sure Keystone is participating
+
+**Meeting Minute Approval**
+
+Proposed: That the minutes of the September 22, 2022 meeting of the Technical Advisory Council of the Confidential Computing Consortium as distributed to the members of the TAC in advance of this meeting are hereby adopted and approved.
+
+   * Quorum was reached and the minutes were approved by the voting members of the TAC. KT to amend if there was a missing action or recommendation from the MPC presentation.
```

Comment:
Was there?
Maybe an action item related to Veracruz?

Reply:
Action item related to Veracruz arose during the 9/22 meeting, Thomas Fossati took an action to reach out to Jordan Brandt of MPC alliance given the large overlap in their respective charters.  Here is the link to the exchange: https://youtu.be/IoUOZZrP_-A?t=4336

Reply:
hi @thomas-fossati the action holder here :-)

I think I said it during last meeting that I had put Derek, Hugo and Jordan in contact around the 23rd of September.  I will check again if there's been any progress on that front, but apart from that the action should be complete.

Reply:
I waited to commit the previous minutes to amend with the changes Alec suggested